content/studies/sample.md

```diff
@@ -24,12 +24,101 @@ This is a pagraph — cillum nisi aliqua officia proident dolor esse sint ut non
+### Side-by-side modules
+div.cols > figure > img + figcaption
 
----
+<div class="cols">
+    <figure>
+        <img src="/img/studies/light-two/sunflower.jpg" alt="">
+        <figcaption>div.cols</figcaption>
+    </figure>
+    <figure>
+        <img src="/img/studies/light-two/phone-square.jpg" alt="">
+        <figcaption>div.cols</figcaption>
+    </figure>
+</div>
+
+<div class="cols items-center">
+    <figure>
+        <img src="/img/studies/light-two/sunflower.jpg" alt="">
+        <figcaption>div.cols.items-center</figcaption>
+    </figure>
+    <figure>
+        <img src="/img/studies/light-two/phone-square.jpg" alt="">
+        <figcaption>div.cols.items-center</figcaption>
+    </figure>
+</div>
+
+<div class="cols items-end">
+    <figure>
+        <img src="/img/studies/light-two/sunflower.jpg" alt="">
+        <figcaption>div.cols.items-end</figcaption>
+    </figure>
+    <figure>
+        <img src="/img/studies/light-two/phone-square.jpg" alt="">
+        <figcaption>div.cols.items-end</figcaption>
+    </figure>
+</div>
+
+<div class="cols">
+    <figure>
+        <img src="/img/studies/light-two/sunflower.jpg" alt="">
+        <figcaption>div with 3 images</figcaption>
+    </figure>
+    <figure>
+        <img src="/img/studies/light-two/phone-square.jpg" alt="">
+        <figcaption>div with 3 images</figcaption>
+    </figure>
+    <figure>
+        <img src="/img/studies/light-two/phone-square.jpg" alt="">
+        <figcaption>div with 3 images</figcaption>
+    </figure>
+</div>
+
+<div class="cols">
+    <figure>
+        <img src="/img/studies/light-two/sunflower.jpg" alt="">
+        <figcaption>div > figure + p</figcaption>
+    </figure>
+    <p>
+        Lorem, ipsum dolor sit <em>amet</em> consectetur adipisicing elit. Molestias cumque cupiditate repellat iusto nulla, nisi sunt pariatur sint voluptatibus. Facilis modi vel debitis, corrupti aliquam itaque rem temporibus soluta ducimus. Lorem, ipsum dolor sit <em>amet</em> consectetur adipisicing elit. Molestias cumque cupiditate repellat iusto nulla, nisi sunt pariatur sint voluptatibus. Facilis modi vel debitis, corrupti aliquam itaque rem temporibus soluta ducimus. Lorem, ipsum dolor sit <em>amet</em> consectetur adipisicing elit. Molestias cumque cupiditate repellat iusto nulla, nisi sunt pariatur sint voluptatibus. Facilis modi vel debitis, corrupti aliquam itaque rem temporibus soluta ducimus. Lorem, ipsum dolor sit <em>amet</em> consectetur adipisicing elit. Molestias cumque cupiditate repellat iusto nulla, nisi sunt pariatur sint voluptatibus. Facilis modi vel debitis, corrupti aliquam itaque rem temporibus soluta ducimus.
+    </p>
+</div>
+
+<div class="cols items-center">
```

Comment:
using basement classes for vertical alignment for image + text, same as side-by-side images